luxonis_ml/data/requirements.txt

```diff
@@ -1,12 +1,12 @@
 numpy>=1.22
 tqdm>=4.64.1
-opencv-python==4.7.0.68
-opencv-python-headless==4.7.0.68
+opencv-python>=4.10.0
+opencv-python-headless>=4.10.0
```

Comment:
Out of curiosity, why did we increase this version?

Reply:
There is a significant improvement in `imread` speed. Version `4.10.0` is ~2 times faster than `4.7.0`